de.metas.adempiere.adempiere/base/src/main/java/de/metas/process/ProcessInfo.java

```diff
@@ -219,6 +222,8 @@ private ProcessInfo(
 	@Getter
 	private final boolean reportApplySecuritySettings;
 	private final OutputType jrDesiredOutputType;
+	@NonNull private final Optional<String> type;
```

Comment:
pls consider using lombok `@Getter`.
The `@NonNull` annotation there is simply pointless.

Reply:
also, what's type? Pls consider having it as enum.

Reply:
@teosarca it's org.compiere.model.I_AD_Process.COLUMNNAME_Type with
org.compiere.model.X_AD_Process.TYPE_JasperReportsSQL